docs/src/pages/roadmap.tsx

```diff
@@ -216,6 +217,12 @@ const roadmap: Item[] = [
 ];
 
 const milestones: Item[] = [
+  // withRelease({
+  //   icon: mdiContentDuplicate,
+  //   title: 'Similar image detection',
+  //   description: 'Detect duplicate assets that aren’t exactly identical',
+  //   release: 'v1.106.0',
+  // }),
```

Comment:
You can actually uncomment this and the block below, ty :)

Reply:
Done :)